src/main/java/de/hysky/skyblocker/utils/Utils.java

```diff
@@ -59,7 +59,7 @@ public class Utils {
      * Current Skyblock location (from the Mod API)
      */
     @NotNull
-    private static Location location = Location.UNKNOWN;
+	private static Location location = Location.UNKNOWN;
```

Comment:
Please don't change this. It's a trivial formatting change and it's the only change in the entire file.